### PR TITLE
build(xtask): :arrow_up: Update XR loaders

### PR DESCRIPTION
**Pull request by zarik5**

### CODE REVIEW

File: alvr/client_openxr/src/lib.rs [hunk 1]
```diff
@@ -121,12 +121,7 @@ pub fn entry_point() {
     let platform = alvr_client_core::platform();
 
     let loader_suffix = match platform {
-        Platform::Quest1
-        | Platform::Quest2
-        | Platform::Quest3
-        | Platform::QuestPro
-        | Platform::QuestUnknown => "_quest",
-        Platform::PicoNeo3 | Platform::Pico4 => "_pico",
+        Platform::Quest1 => "_quest1",
```
Comment (The-personified-devil) [inline alvr/client_openxr/src/lib.rs:124]: We wanted to keep the pico loader

Comment (zarik5) [reply alvr/client_openxr/src/lib.rs:124]: fixed

File: alvr/client_openxr/src/lib.rs [hunk 2]
```diff
@@ -139,6 +135,7 @@ pub fn entry_point() {
     xr_entry.initialize_android_loader().unwrap();
 
     let available_extensions = xr_entry.enumerate_extensions().unwrap();
+    alvr_common::info!("OpenXR available extensions: {available_extensions:#?}");
```
Comment (The-personified-devil) [inline alvr/client_openxr/src/lib.rs:138]: Especially this one I would expect to be quite spammy so it should probably be a debug log instead

**Review by Meister1593 — APPROVED** [review]

File: alvr/client_openxr/src/lib.rs [hunk 3]
```diff
@@ -167,6 +164,9 @@ pub fn entry_point() {
     exts.khr_convert_timespec_time = true;
     exts.khr_opengl_es_enable = true;
 
+    let available_layers = xr_entry.enumerate_layers().unwrap();
+    alvr_common::info!("OpenXR available layers: {available_layers:#?}");
```
Comment (Meister1593) [inline alvr/client_openxr/src/lib.rs:168]: this could be spammy too

Comment (zarik5) [reply alvr/client_openxr/src/lib.rs:168]: it outputs 0 layers

File: alvr/client_openxr/src/lib.rs (hunk 2 above)
Comment (zarik5) [reply alvr/client_openxr/src/lib.rs:138]: This is logged before connection, so it's just printed to logcat

Comment (zarik5) [reply alvr/client_openxr/src/lib.rs:138]: The Quest runtime already prints the list before, but our log is a little cleaner

**Review by The-personified-devil — APPROVED** [review]

**Pull request merged**